core/src/main/java/org/infinispan/interceptors/impl/BaseStateTransferInterceptor.java

```diff
@@ -25,11 +28,10 @@
  * Also, it has some utilities methods with the most common logic.
  *
  * @author Pedro Ruivo
- * @deprecated Since 8.2, no longer public API.
+ * @since 9.0
  */
-@Deprecated
-public abstract class BaseStateTransferInterceptor extends CommandInterceptor {
-   private final boolean trace = getLog().isTraceEnabled();
+public abstract class BaseStateTransferInterceptor extends DDSequentialInterceptor {
+   private static final Log log = LogFactory.getLog(BaseStateTransferInterceptor.class);
```

Comment:
I prefer the old log scheme. I mean, I prefer to see logs from a single state transfer interceptor than have to look for the abstract and concrete class.